### DISCUSSION TITLE
[SECURITY] Your Issue Title

### DISCUSSION THREAD

**Discussion by WTFHello** [nextgenhealthcare/connect#4835] [General]
I did not install the beta version or this package on my text now app thru Google (play)
https://beta.pithus.org/report/7616d080b6c57c7e939e9d324ffb259c07185c961ffd7c1ac858beb72250ea55
*Vulnerability Location**
Google Play, Chrome, phone software

**Environment (please complete the following information if it is applicable to the issue)** my phone is running on Mozilla Mac os that I never installed. and I have an android LG. my phone is being cloned.
 

**Suggested remediation*
 Update cert and to correct version. remove your package. I can not get my calls or texts.

I also have this cert that was attached. Please update.
Network locked

all 66%

7:13 PM

Certificate viewer

www.github.com

Organizational Unit (OU) www.digicert.com

VALIDITY PERIOD

Issued On

May 5, 2020

Expires On

Apr 14, 2022

FINGERPRINTS

SHA-256 Fingerprint

61 79 D2 FB 1F F1 DA 04 E8 D2 8B 28 BA 9F 84 CB 24 B4 B8 BC B8 CO E2 5B 31 25 1D 67 87 80 DO 89

SHA-1 Fingerprint

70 94 DE DD E6 C4 69 48 3A 92 70 A1 48 56 78 2D 18 64 €0 B7

EXTENSIONS

Certificate Subject Alternative

Name

www.github.com

e's

Ses

**Comment by WTFHello** [comment]
sorry, if it's not written properly. I just really need my text now app so I can able to communicate. my notifications and access to the app is being blocked.

**Comment by pacmano1** [comment]
I don't think you are in the right place.   This location is for a product called mirth connect used for healthcare integration.  It has nothing to do with phones or Google play.

**Comment by WTFHello** [comment]
well then there are more problems because that is where your packed linked
when I clicked "contact us"

jessica marie olson

**Comment by pacmano1** [comment]
This open source project has NOTHING to do with Andriod or mobile phones at all.

Who is "your" in your sentence?   What packed link? 

What _contact us_ link are you talking about?  Post the web page where you saw that.    

I am responding your post as a courtesy, take the time to be as courteous please.  I don't work for the company that supports this project, I am just a user of mirth conenct.   Please _read_ https://www.nextgen.com/products-and-services/integration-engine, you will see it has nothing to do with what you posted.

**Reply by pacmano1** [reply]
This has been moved to discussions, since it does not appear the orginal poster reaiizes they are in the wrong place.
